context_builder.Dockerfile

```diff
@@ -1,18 +1,19 @@
 # Builds baseten/truss-context-builder, a light-weight image that can be used
 # for creating docker build context out of a Truss.
 # Build that image as:
-# docker buildx build . -f context_builder.Dockerfile --platform=linux/amd64 -t baseten/truss-context-builder
+# docker buildx build . -f context_builder.Dockerfile --platform=linux/amd64,linux/arm64 -t baseten/truss-context-builder
 FROM python:3.9-slim
 
 RUN apt-get update \
     && apt-get install --yes --no-install-recommends curl tar \
     && apt-get autoremove -y \
     && apt-get clean -y \
+    && apt-get install -y build-essential \
     && rm -rf /var/lib/apt/lists/* /tmp/library-scripts/
 
-RUN curl -sSL https://install.python-poetry.org | python -
-
-ENV PATH="/root/.local/bin:${PATH}"
+RUN curl -sSL https://install.python-poetry.org | python - \
```

Comment:
I'm guessing you're adding rust since it's needed to build a downstream python dependency? Maybe its worth adding a comment somewhere?